src/index.js

```diff
@@ -717,6 +719,10 @@ function applyMeasureModifiers({
         }
         availableModifiers[modifier.type].initModifier(modifier);
 
+        if (modifier.type === 'timeSeriesDecomposition' && modifier.decomposition === '') {
```

Comment:
This logic should be moved to inside initModifier in the time-series-decomposition/index.js file

Reply:
Made  requested change, thanks for the review!